app/Modules/Group/Events/GroupStatusUpdated.php

```diff
@@ -23,6 +23,11 @@ class GroupStatusUpdated extends GroupEvent
      */
     public function __construct(public Group $group, public GroupStatus $newStatus, public GroupStatus $oldStatus)
     {
+
+        // Only process the event if the status is 2 or 5
+    if (!in_array($this->group->group_status_id, [2, 3, 4, 5])) {
+        return;
```

Comment:
Does this even do anything different? It looks to just be returning from the constructor "early", but it would also return right after this statement.